packages/react/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -0,0 +1,62 @@
+import { IComboBoxProps } from '../../ComboBox';
+
+/**
+ * Range of start and end hours to be shown in the TimePicker.
+ */
+export interface ITimeRange {
+  /** Start hour (inclusive) for the time range, 0-23 */
+  start: number;
+  /** End hour (exclusive) for the time range, 0-23 */
+  end: number;
+}
+
+/**
+ * Localized strings to use in the TimePicker
+ */
+export interface ITimePickerStrings {
+  /** Error message to render for TextField if input parsing fails. */
+  invalidInputErrorMessage: string;
+}
+
+export interface ITimePickerProps extends Omit<IComboBoxProps, 'options'> {
```

Comment:
You should also omit a few more ComboBox props that don't work with the design of the TimePicker:
```suggestion
export interface ITimePickerProps extends Omit<IComboBoxProps, 'options' | 'selectedKey' | 'defaultSelectedKey' | 'multiSelect' | 'text'> {
```

Reply:
Resolved by updating Omit fields to 
`'options' | 'selectedKey' | 'defaultSelectedKey' | 'multiSelect' | 'text'`.